ibrahimjaved12/ocw-hugo-themes
# Lighten stories text

#### Pre-Flight checklist


#### What are the relevant tickets?
Corresponds to 939

#### What's this PR do?
Previously, a darkening linear gradient was applied to the whole image of stories/testimonials. But this change makes it such that the gradient is only applied within the elements containing the text.

#### How should this be manually tested?
Checkout to this branch, run the code and observe if the gradient is applied only on text or whole images.

#### Screenshots (if appropriate)
Before:
<img width="1375" alt="Screenshot 2022-12-28 at 5 10 47 PM" src="https://user-images.githubusercontent.com/109785089/209824466-b6aa6d95-5066-4d23-b8ef-c089ee15621d.png">

After:
<img width="1276" alt="image" src="https://user-images.githubusercontent.com/109785089/209824535-a4910ef5-8f59-4204-ae4c-40d8d014d148.png">
